docs/helpers/Triggers.md

```diff
@@ -0,0 +1,109 @@
+---
+title: State Triggers
+author: dotMorten
+description: A collection of custom visual State Triggers
+keywords: windows 10, uwp, windows community toolkit, uwp community toolkit, uwp toolkit, CompareStateTrigger, FullScreenModeStateTrigger, IsEqualStateTrigger, IsNotEqualStateTrigger, IsNullOrEmptyStateTriggers, NetworkConnectionStateTrigger, RegexStateTrigger, UserHandPreferenceStateTrigger, UserInteractionModeStateTrigger
+
+---
+
+# State Triggers
+
+<!-- Describe your control -->
+A collection of custom visual [State Triggers](https://docs.microsoft.com/dotnet/api/Microsoft.Toolkit.Uwp.UI.Triggers)
```

Comment:
In this case, let's actually link to the system api docs as that's the base class (and each has a link below to our api doc)

```suggestion
A collection of custom visual [State Triggers](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.statetrigger).
```